spaces/S000019/properties/P000142.md

```diff
@@ -0,0 +1,7 @@
+---
+space: S000019
+property: P000142
+value: false
+---
+
+Suppose that $K\subseteq X$ is compact Hausdorff. Then $K\cap [a, b]$ is a compact subset of $[a, b]\subseteq X$, where the topology on $[a, b]$ coincides with Euclidean topology. It follows that $K$ is closed in the Euclidean topology. If $K$ were unbounded, and if $K_1, K_2\subseteq K$ are compact sets in the Euclidean topology, then $(K\setminus K_1)\cap (K\setminus K_2)\neq\emptyset$ so that $K$ is not Hausdorff in the subspace topology coming from $X$. It follows that compact Hausdorff subsets of $X$ are precisely the subsets compact in Euclidean topology. The topology which those sets determnine is the Euclidean topology and not the topology of $X$, so that $X$ is not a $k_3$-space.
```

Comment:
```suggestion
Suppose that $K\subseteq X$ is compact Hausdorff. Then $K\cap [a, b]$ is a compact subset of $[a, b]\subseteq X$, where the topology on $[a, b]$ coincides with the Euclidean topology. It follows that $K$ is closed in the Euclidean topology. If $K$ were unbounded, and if $K_1, K_2\subseteq K$ are compact sets in the Euclidean topology, then $(K\setminus K_1)\cap (K\setminus K_2)\neq\emptyset$ so that $K$ is not Hausdorff in the subspace topology coming from $X$. It follows that compact Hausdorff subsets of $X$ are precisely the subsets compact in the Euclidean topology. The topology which those sets determine is the Euclidean topology (since {S25|P140}) and not the topology of $X$, so that $X$ is not a $k_3$-space.
```